feat: View Hierarchy spec update (#6091)
getsentry/sentry-docs: src/platforms/common/enriching-events/viewhierarchy.mdx

  - dotnet.xamarin
---

> When a user experiences an error, Sentry provides the ability to render a JSON represention of your view hierarchy and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.

Reviewer: ```suggestion
Sentry makes it possible to render a JSON representation of the view hierarchy of an error and includes it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.
```
